src/collision.py

```diff
@@ -0,0 +1,51 @@
+from __future__ import annotations
+
+import arcade
+
+from bullet import Projectile
+from linked_sprite import LinkedSprite
+from player import Player
+
+
+def projectile_hits_wall(projectile_spritelist, wall_spritelist):
```

Comment:
You can pass this function `sprite_lists` as the argument, and you can call `sprite_lists.projectiles` and `sprite_lists.walls`